aten/src/ATen/core/ivalue.h

```diff
@@ -186,6 +187,10 @@ struct CAFFE2_API IValue final {
     return payload.as_double;
   }
 
+  IValue(ScalarType t) : tag(Tag::ScalarType) {
```

Comment:
nothing else in this file is constructed in this way -- they all set is_intrusive_ptr(false) as well.  Why the difference here?

Reply:
yup this was a mistake